modules/extensions/apps/portal/store/carbon.super/fs/gadget/task_instance_count_vs_date/index.xml

```diff
@@ -90,21 +85,14 @@
 </head>
 
 <body>
-    <div class="panel panel-default">
-        <div class="panel-body">
+    <div>
+        <div class="panel-body" style="width:100%; height:95%">
```

Comment:
This is fixed by https://github.com/wso2/product-pc/commit/e931bb704e9065cf57f08f61619a45a4beecb5c7